### PR TITLE
Add Severity filter for annotations output.

### PR DESCRIPTION
**Pull request by michael-sicpa**
Add severity filter for annotations output on github actions.
The fact that all vulnerabilities are displayed make this github actions very hard to use.
I added testing too.
Thank you
Michael

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,11 +57,21 @@ It might be required when scanning from docker-daemon or cri-o to provide a user
...
-
 ### `extra-docker-parameters`
 
 Additional parameters added to the `docker` command when executing the secure-inline-scan container execution.
 
+### `severity`
+
+Filter output annotations by severity. Default is "unknown".
+Possible values:
+- ciritcal
```
Comment (derlin) [inline README.md:68]: typo: ` ciritcal` --> `critical` (here and in the source files)

File: README.md [hunk 2]
```diff
@@ -57,11 +57,21 @@ It might be required when scanning from docker-daemon or cri-o to provide a user
...
+- high
+- medium
```
[the remark this answers is not in the document]
Comment (michael-sicpa) [reply README.md:70]: This is documented on Sysdig official page:
[https://docs.sysdig.com/en/docs/sysdig-secure/scanning/review-scan-results/scan-result-details/](https://docs.sysdig.com/en/docs/sysdig-secure/scanning/review-scan-results/scan-result-details/)

> Search or filter by severity: Critical, High, Medium, Negligible, Unknown. Also filter by whether it “Has a Fix”.

File: README.md (hunk 1 above)
Comment (michael-sicpa) [reply README.md]: Fixed

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (michael-sicpa) [reply README.md:71]: I added low severity, as it is a valid output from the build logs, even through it is missing in the doc.

**Comment by michael-sicpa** [comment]
@airadier Can you review it please?
Thanks

**Review by airadier — APPROVED** [review]
Thanks very much for the contribution!!

**Comment by michael-sicpa** [comment]
Do you have an idea of when this PR can be merged?

Thank you

**Comment by michael-sicpa** [comment]
@airadier Do you have the permissions to merge?
I have another PR I would like to propose you :)
Thanks

**Pull request merged**